### ISSUE TITLE
Radar Guided Missiles will not track

### ISSUE THREAD

**Issue by Isenderen** [PapaJoesSoup/BDArmory#637]
##### ISSUE TYPE
 - Bug Report   (Something is broken)

##### KSP and BDA Version

```KSP 1.4.4.2215

```BDAc 1.2.2.2

##### OS / ENVIRONMENT

Windows 10 64 bit

##### SUMMARY
Radar guided missiles instead of tracking the target are going into a shallow (~8 degree) dive.

##### STEPS TO REPRODUCE
set up 2 planes, one with a radar and a radar guided missile and the other as a target a good distance away (ive been doing it at 100km+), then try to fire the missile at the target plane.


##### Log Files

[KSP.log](https://github.com/PapaJoesSoup/BDArmory/files/2776345/KSP.log)

**Comment by meltingSnowdrift** [comment]
I suspect this is due to the missiles trying to get to the target through the ground due to the curvature of Kerbin. If that is the case, I have also often had such a problem. It considerably and unrealistically restricts the design of SAM systems.

To make things worse, guard mode, in such situations, often tries to fire missiles at distant targets causing the missiles to end up hitting the ground or ocean.

**Comment by xdthegreat** [comment]
1. Is there a lock?
2. Is the missile designed for firing up to 100km?

**Comment by xdthegreat** [comment]
And v 1.2.2.2 is ancient.

**Comment by Isenderen** [comment]
Yes to both and It looks like i forgot to type it but im still on KSP1.4.4. Thats why im still using 1.2.2.2.

**Comment by Isenderen** [comment]
So ive been doing more testing at closer ranges and it looks to me like a lofting profile was added for the missiles to follow beyond a certain range (roughly 50km from what im seeing) but the missile is following it upsidedown.

In a couple of tests firing at roughly 60km ive just done the missile goes into a roughly -5 to -7 degree dive, levels off after around 10 seconds, continues to boost until roughly 20km, then it starts a gentle turn to the target, before yanking a rediculous amount of G forces at around 5km trying to get a 70+ degree turn to intercept at the last second destroying the missile.

**Comment by SpannerMonkey** [comment]
HI  as the issue doesn't appear to be present in later versions  it's doubtful if any resolution for your issue can be realistically diagnosed or rectified .  Bug reports and bug fixes tend to be  specific to builds  unless a problem persists through many versions, even  in that case it's  debatable if it's worth spending the personal time on fixing an issue in a long past version  for what must be an ever shrinking pool of users.
